### PR TITLE
Update french tutorial translation for the V3

### PR DESCRIPTION
**Pull request by framatophe**
Update, modifications and fixes

### CODE REVIEW

File: static/tutorial/fr/welcome.md [hunk 1]
```diff
@@ -23,61 +23,64 @@ Mais, maintenant : Allons-y !
...
 
 Bien sûr, il y a beaucoup d'autres éléments. Les notes de bas de page par exemple — survolez celui-ci avec votre curseur[^1]. Au cours de ce tutoriel, vous apprendrez également certains éléments spéciaux que Zettlr utilise pour permettre un travail véritablement académique, ainsi que la gestion des connaissances à l'aide des Zettelkästen !
 
 ## Liens ⛓
 
-Bien qu'ils ne soient pas très souvent utilisés dans les textes académiques, les liens constituent un outil puissant de Markdown, et Zettlr les fait passer au niveau supérieur. Zettlr agit de manière très intelligente en ce qui concerne les liens. Rapidement, créons-en un ! Sélectionnez le lien suivant vers notre compte Twitter, et copiez-le dans le presse-papiers : https://www.twitter.com/Zettlr
+Bien qu'ils ne soient pas très souvent utilisés dans les textes académiques, les liens constituent un outil puissant de Markdown, et Zettlr les fait passer au niveau supérieur. Zettlr agit de manière très intelligente en ce qui concerne les liens. Rapidement, créons-en un ! Sélectionnez le lien suivant vers notre compte Twitter, et copiez-le dans le presse-papiers : https://fosstodon.org/@zettlr
```
Comment (nathanlesage) [inline static/tutorial/fr/welcome.md:54]: I'm personally in to include the Mastodon account here, but then it should also read "Sélectionnez le lien suivant vers notre compte Mastodon,..." right?

**Review by nathanlesage — APPROVED** [review]
Looks good, only one time you kept "Twitter" while replacing the Twitter link with the Mastodon link 😅

**Comment by nathanlesage** [comment]
Ah, and apparently you forgot to merge some changes from the develop branch which now prevents merging. Since you mentioned you're not too good with Git, feel free to ping me so I can clear them for you.

File: static/tutorial/fr/welcome.md (hunk 1 above)
Comment (framatophe) [reply static/tutorial/fr/welcome.md:54]: Yes! thanks :) : « vers notre compte Mastodon » (sorry)

**Comment by framatophe** [comment]
Yes, thank you! Please delete it. I didn't quite understand what it was about. I have left it in doubt....

**Comment by nathanlesage** [comment]
Okay, somehow I can't really resolve the problems. Sorry! I think we need to do the following:

1. Copy all the modified tutorial files to some safe place on your computer
2. Delete this PR, delete your fork, and then fork the entire repository anew
3. Paste the modified tutorial files again into your fork, and open a new PR

This way it should work.

**Comment by framatophe** [comment]
OK. Sorry. I will do it before 11: am :)

**Pull request closed without merging**